### PR TITLE
Update license header infos

### PR DESCRIPTION
**Pull request by AndyScherzinger**
... to be in line with SPDX header strategy

### ☑️ Resolves

* Fix none

### CODE REVIEW

File: developer_manual/getting_started/codingguidelines.rst [hunk 1]
```diff
@@ -71,27 +71,35 @@ Nextcloud is licensed under the `GNU AGPLv3 <https://www.gnu.org/licenses/agpl>`
...
+  /**
+   * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+   * SPDX-License-Identifier: AGPL-3.0-or-later
+   */
+
+The Nextcloud GmbH part only applies to employees of the company not to contributors.
+
+For more, general information on SPDX headers, please see 
+
+* https://reuse.software/
```
Comment (ChristophWurst) [inline developer_manual/getting_started/codingguidelines.rst:101]: does this render as link or text?

Comment (AndyScherzinger) [reply developer_manual/getting_started/codingguidelines.rst]: fixed in https://github.com/nextcloud/documentation/pull/11993/commits/07e3627cfca90c45c5785a26fab855e0318ada8c

File: developer_manual/getting_started/codingguidelines.rst [hunk 2]
```diff
@@ -71,27 +71,35 @@ Nextcloud is licensed under the `GNU AGPLv3 <https://www.gnu.org/licenses/agpl>`
...
+  /**
+   * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+   * SPDX-License-Identifier: AGPL-3.0-or-later
+   */
+
+The Nextcloud GmbH part only applies to employees of the company not to contributors.
+
+For more, general information on SPDX headers and their usage for reuse-compliance, please see 
+
+* `REUSE <https://reuse.software/>`
```
Comment (ChristophWurst) [inline developer_manual/getting_started/codingguidelines.rst:101]:
```suggestion
* `REUSE <https://reuse.software/>_`
```

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

did not test but other links on this patch have a trailing underscore too.

Comment (AndyScherzinger) [reply developer_manual/getting_started/codingguidelines.rst]: Fixed: https://github.com/nextcloud/documentation/pull/11993/commits/701cf5111b4fc6a8c508006bb2d52cfe6457ac6c

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**